src/simulation/environment/MsisAtmosphere/_UnitTest/test_msiseAtmosphere.py

```diff
@@ -219,17 +219,17 @@ def run(show_plots, orbitCase, setEpoch):
     #   Test atmospheric density calculation; note that refAtmoData is in g/cm^3,
     #   and must be adjusted by a factor of 1e-3 to match kg/m^3
     print(densData[-1])
-    print(refAtmoData[5])
+    print(refAtmoData[5]*1000)
     if not unitTestSupport.isDoubleEqualRelative(densData[-1], refAtmoData[5]*1000., accuracy):
         testFailCount += 1
-        testMessages.append("FAILED:  NRLMSISE-00 failed density unit test with a value difference of "+str(densData[0]-refAtmoData[5]))
+        testMessages.append("FAILED:  NRLMSISE-00 failed density unit test with a value difference of "+str(densData[0]-refAtmoData[5]*1000))
```

Comment:
:large_orange_diamond: Seeing as we are touching the tests here, we might benefit by migrating this test to use the `np.testing` asserts in this file.

Reply:
I found this link which could be useful: https://numpy.org/doc/stable/reference/routines.testing.html

Reply:
I've updated the test to use `np.testing`.